integration/test/performancetest/performance_test.go

```diff
@@ -1,22 +1,28 @@
-// go:build linux && integration
+//go:build linux && integration
 // +build linux,integration
 
 package performancetest
 
-import(
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"sync"
 	"testing"
 	"time"
-	"log"
-	"context"
+
 	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 const (
-	configPath = "resources/config.json"
-	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
-	agentRuntimeMinutes = 5 //20min desired but 5mins for testing purposes 
-	DynamoDBDataBase = "CWAPerformanceMetrics"
-
+	configPath               = "resources/config.json"
+	configOutputPath         = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	agentRuntimeMinutes      = 5 //20 mins desired but 5 mins for testing purposes
+	DynamoDBDataBase         = "CWAPerformanceMetrics"
+	logOutputPath1           = "/tmp/test1.log"
+	logOutputPath2           = "/tmp/test2.log"
+	transactionRatePerSecond = 10
```

Comment:
Nitpick - we don't need to be so verbose with the name. I think tps is acceptable

Reply:
Updated